Yoast/wordpress-seo
# Deprecate all SEO score recalculation related code

## Context


* In 2016 [(see issue)](https://github.com/Yoast/wordpress-seo/issues/4532) we've 'temporary' disabled the recalculation tool for recalculating the SEO scores. A lot has been changed in the meanwhile and I don't expect that we will reintroduce this feature shortly. See also this [pull request](https://github.com/Yoast/wordpress-seo/pull/14838) that removes related notification code.

## Summary


This PR can be summarized in the following changelog entry:

* N/A - Deprecation of code that isn't used.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Test if everything still works: `grunt build` should build normally and reindexation of the links should still work.
* Test in premium as well

## UI changes

## Documentation

## Quality assurance


Fixes #
